ncm-metaconfig/src/main/metaconfig/udev/blockdev.tt

```diff
@@ -0,0 +1,8 @@
+[%- FOREACH dv IN scsi_run -%]
+SUBSYSTEM=="block", SUBSYSTEMS=="scsi", PROGRAM=="/lib/udev/scsi_id -g -u -d /dev/%k", RUN+="[% dv  %]"
```

Comment:
the magic script first loops over devices and checks if /sys/block/$dev/device/model is a MD34xx to not include eg boot disk.